policyengine_us/parameters/gov/hhs/chip/FCEP/income_limit.yaml

```diff
@@ -0,0 +1,111 @@
+description: Maximum family income for pregnant people to qualify for CHIP (FCEP).
```

Comment:
Could you change to active voice for the descriptions

Also could you write out CHIP in those
```
[Responsible department] limits the Children's Health Insurance Program to families with pregnant members with income below this percentage of federal poverty guidelines, based on state.

Reply:
Something like this for all descrptions